pkgs/top-level/all-packages.nix

```diff
@@ -29420,11 +29420,11 @@ with pkgs;
 
   scli = callPackage ../applications/misc/scli { };
 
-  scribus = callPackage ../applications/office/scribus {
+  scribus_1_4 = callPackage ../applications/office/scribus/1_4.nix {
     inherit (gnome2) libart_lgpl;
   };
 
-  scribusUnstable = libsForQt5.callPackage ../applications/office/scribus/unstable.nix { };
```

Comment:
This should at least get an entry in `pkgs/top-level/aliases.nix`.

Reply:
Can I make the alias warn somehow? (This usually leads to eval errors, but maybe there is a cleverer way.)

Reply:
I added a throw alias now, is that what you wanted?

Reply:
Yes, I think that's the right way. Thanks.